tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -760,36 +759,33 @@ func (az *AzureProvider) CreateWindowsVM() (*types.Credentials, error) {
 	}
 	vmInfo, err := future.Result(az.vmClient)
 	if errorCheck(err) {
-		log.Error(err, fmt.Sprintf("failed to obtain instance info"))
+		log.Printf("failed to obtain instance info: %s", err)
 	}
 
 	resourceTrackerFilePath, err := resource.MakeFilePath(az.resourceTrackerDir)
 	if errorCheck(err) {
-		log.Error(err, fmt.Sprintf("unable to create a file"))
+		log.Printf("unable to create resource file: %s", err)
 	}
 
 	err = resource.AppendInstallerInfo([]string{*(vmInfo.Name)}, []string{az.NsgName}, resourceTrackerFilePath)
 	if errorCheck(err) {
-		log.Error(err, fmt.Sprintf("unable to append the resources"))
+		log.Printf("unable to add installer info to the resource file: %s", err)
 	}
 
-	log.Info(fmt.Sprintf("Successfully created windows instance: %s", instanceName))
-
-	ipAddress, ipErr := az.getIPAddress(ctx)
-	if errorCheck(ipErr) {
-		log.Error(err, fmt.Sprintf("couldn't get the IP Address for corresponding resource: %s", az.IpName))
+	ipAddress, err := az.getIPAddress(ctx)
+	if errorCheck(err) {
+		log.Printf("failed to get the IP address of %s: %s", az.IpName, err)
 		*ipAddress = az.IpName
 	}
 	resultData := fmt.Sprintf("xfreerdp /u:core /v:%s /h:1080 /w:1920 /p:'%s' \n", *ipAddress, adminPassword)
 	resultPath := az.resourceTrackerDir + instanceName
 	err = resource.StoreCredentialData(resultPath, resultData)
 	if errorCheck(err) {
-		log.Error(err, fmt.Sprintf("unable to write data into file"))
-		log.Info(fmt.Sprintf("xfreerdp /u:core /v:%s /h:1080 /w:1920 /p:%s", *ipAddress, adminPassword))
-	} else {
-		log.Info(fmt.Sprintf("Please Check for file %s in %s directory on how to access the node",
-			instanceName, az.resourceTrackerDir))
+		log.Printf("unable to write data into resource file: %s", err)
+		log.Printf("xfreerdp /u:core /v:%s /h:1080 /w:1920 /p:%s", *ipAddress, adminPassword)
 	}
+	log.Printf("Please Check for file %s in %s directory on how to access the node",
```

Comment:
This needs to be in the `else` otherwise you are asking the user to look for a file that has no data in it.